finesse/scripts/xlsx_converter_json.py

```diff
@@ -0,0 +1,56 @@
+import openpyxl
+import os
+import json
+import argparse
+
+parser = argparse.ArgumentParser(description='XLSX Converter to JSON')
+parser.add_argument('--input-folder', dest='input_folder', default='finesse/scripts/input/', help='Path to the input folder')
+parser.add_argument('--output-folder', dest='output_folder', default='finesse/scripts/output/', help='Path to the output folder')
+parser.add_argument('--file-name', dest='file_name', help='Name of the input file')
+parser.add_argument('--sheet-name', dest='sheet_name', default='To fill', help='Name of the worksheet')
+
+args = parser.parse_args()
+
+INPUT_FOLDER = args.input_folder
+OUTPUT_FOLDER = args.output_folder
+FILE_NAME = args.file_name
+SHEET_NAME = args.sheet_name
+FILE_PATH = INPUT_FOLDER+FILE_NAME

```

Comment:
this assume that the input folder ends with a /

better to use os.path.join to ensure platform-independant file joining.

Reply:
Done

Reply:
Note that reviewer should resolve the conversation, not the author.